### ISSUE TITLE
[Bug]: Context Big problems with the new ComfyUI frontend

### ISSUE THREAD

**Issue by gisu** [rgthree/rgthree-comfy#292]
Hi,
I use Big Nodes in my workflow context. The node generates several ZOD schema errors in the new UI. The workflow is then no longer displayed (I have already written something about this in the Comfy Repo because debugging is difficult).

![image](https://github.com/user-attachments/assets/76916d1e-4ea4-480a-990c-0386ac91f860)

You can easily reproduce this by starting the current ComfyUI version with the new frontend. It is sufficient to drop the Context Big Node into the workflow and reload the browser. 

The smaller Context Node has no problems, but I cannot say which node is still affected. Theoretically, all nodes that generate the frontend via JS are possible.

**Comment by alessandroperilli** [comment]
I had a similar problem with AP Workflow, but I didn't realize that the culprit was the Context Node. Are you sure?

Regardless: I worked with the developers and, since version 1.2.18 of the new frontend, you shouldn't see this error anymore. They relaxed the validation.

**Comment by gisu** [comment]
I was sure, I tried this for several hours with try & error (delete nodes step by step from the existing workflow, close old ui, start new ui). In the end I had an empty workflow with only one context big node in it and that's where the error was. But I just did the test (had not started the new frontend for several days because of the problems). 1.2.21 and the workflow can be opened without any problems.
